tests/test_fyle/test_tasks.py

```
@@ -490,3 +502,966 @@ def test_skip_expenses_and_post_accounting_export_summary(mocker, db):
+
+    assert mock_handle_fund_source.call_count == 1
+    assert mock_logger.exception.call_count == 1
+    mock_logger.exception.assert_called_with(
+        "Error handling expense fund source change for workspace_id: %s, report_id: %s | ERROR: %s",
+        workspace.id, report_id, mock_handle_fund_source.side_effect
+    )
+
+
+
+
+def test_handle_expense_fund_source_change_complete_flow(mocker, db):
```

Ruff (0.13.1)

Unused function argument: `db`

(ARG001)